setup.cfg

```diff
@@ -16,6 +16,10 @@ ignore =
   # black loves the double quotes
   Q000
 
+per-file-ignores =
+  # it is possibble to have prints in scripts
+  python-project-lvl1/brain_games/scripts/*.py: WPS421
```

Comment:
flake8 will be executed from project core folder, so there is no need for `python-project-lvl1`  

just keep `brain_games/scripts/*.py: WPS421`